lib/content/audio/following_train.ex

```diff
@@ -48,31 +48,14 @@ defmodule Content.Audio.FollowingTrain do
   defimpl Content.Audio do
     alias PaEss.Utilities
 
-    @the_following "667"
-    @train_to "507"
-    @in_ "504"
-    @minutes "505"
-    @minute "532"
-
     def to_params(audio) do
-      dest_var = Utilities.destination_var(audio.destination)
-
-      if Utilities.directional_destination?(audio.destination) do
-        do_ad_hoc_message(audio)
-      else
-        green_line_branch = Content.Utilities.route_branch_letter(audio.route_id)
-
-        cond do
-          !is_nil(green_line_branch) ->
-            green_line_with_branch_params(audio, green_line_branch, dest_var)
-
-          audio.minutes == 1 ->
-            {:canned, {"159", [dest_var, verb_var(audio)], :audio}}
+      min_or_mins = if(audio.minutes == 1, do: :minute, else: :minutes)
```

Comment:
I am still learning, so I'm not 100% sure if this is correct or necessary, but could the same thing be achieved more functionally with pattern matching like so:
```  
def min_or_mins(%{minutes: 1}), do: :minute

def min_or_mins(%{minutes: _}), do: :minutes
  ```
  
 and then it could be called with `min_or mins(audio)` here and later in this file.

Reply:
Yes, that would work, although the code below is slightly different because it returns strings instead of atoms, so it's not quite reusable in this case.

More generally, I think pattern matching can be a great tool in a lot of situations, but it depends on the context what the clearest option is. I picked `if` here because there are just two branches, and the logic is nice and compact. `case` really shines when there are multiple branches that are logically related. Multi-headed functions like that tend to work well when the branches are more logically disjoint. And of course re-usability plays a part too.

Reply:
Gotcha, thanks for explaining your thought process here! I understand how `if` makes the most sense here then